client/src/ui/Statistics.tsx

```diff
@@ -35,107 +35,127 @@ export default class Statistics extends Component<Props,State>{
     //{cs: [{date1:totalNum}, {date2: totalNum}, ...], math: [{date1:total}, {date2: total}, ...] }
-      Meteor.call('getReviewsOverTimeTop15', Session.get("token"), this.state.step, this.state.range, (err:any, res:any)=>{
-        let data:any[]=[];
-        //key-> EX: cs
-        for(let key in res){
-          let finalDateObj:any={};//{date1:totalNum, date2:totalNum}
-          let obj:any ={}; // {name: cs, data: {date1:totalNum, date2:totalNum}}
-          obj.name=key;
-
-          //[{date1:totalNum}, {date2: totalNum}, ...]
-          let arrDates = res[key];
-
-          arrDates.forEach((arrEntry:any)=>{
-            let dateObject = Object.keys(arrEntry); //[date1]
-            dateObject.forEach(date=>{
-              finalDateObj[date]=arrEntry[date]
-            });
+    Meteor.call('getReviewsOverTimeTop15', Session.get("token"), this.state.step, this.state.range, (err: any, res: any) => {
+      let data: any[] = [];
+      //key-> EX: cs
+      for (let key in res) {
+        let finalDateObj: any = {};//{date1:totalNum, date2:totalNum}
+        let obj: any = {}; // {name: cs, data: {date1:totalNum, date2:totalNum}}
+        obj.name = key;
+
+        //[{date1:totalNum}, {date2: totalNum}, ...]
+        let arrDates = res[key];
+
+        arrDates.forEach((arrEntry: any) => {
+          let dateObject = Object.keys(arrEntry); //[date1]
+          dateObject.forEach(date => {
+            finalDateObj[date] = arrEntry[date]
           });
+        });
 
-          obj.data=finalDateObj;
-          data.push(obj);
-        }
+        obj.data = finalDateObj;
+        data.push(obj);
+      }
 
-        this.setState({chartData: data});
-      });
+      this.setState({ chartData: data });
+    });
   }
 
-  howManyReviewsEachClass(){
-    Meteor.call('howManyReviewsEachClass', Session.get("token"), (error:any, result:any) =>{
-      if(error === null){
+  howManyReviewsEachClass() {
+    Meteor.call('howManyReviewsEachClass', Session.get("token"), (error: any, result: any) => {
+      if (error === null) {
         //sort descending
-        result.sort((rev1:any, rev2:any)=>(rev1.total > rev2.total)?-1:1);
-        this.setState({howManyReviewsEachClass: result});
-      } else{
-          console.log(error);
+        result.sort((rev1: any, rev2: any) => (rev1.total > rev2.total) ? -1 : 1);
+        this.setState({ howManyReviewsEachClass: result });
+      } else {
+        console.log(error);
       }
     });
   }
 
-  howManyEachClass(){
-    Meteor.call('howManyEachClass', Session.get("token"), (error:any, result:any) =>{
-      if(error === null){
-        result.sort((rev1:any, rev2:any)=>(rev1.total > rev2.total)?-1:1);
-        this.setState({howManyEachClass: result});
-      }else{
+  getReviewsPerClassCSV() {
+    let strRet = "class,total\n";
+    this.state.howManyReviewsEachClass.map((obj) => {
+      strRet += obj._id + "," + obj.total + "\n";
+    });
+    return strRet;
+  }
+
+  howManyEachClass() {
```

Comment:
A potential minor change that could help with clarity is changing the name of the howManyEachClass function in Statistics.tsx since there's also a field in state with the same name, otherwise code looks good.